fusion/Youtube.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Youtube = ({id, width, height}) =>(
```

Comment:
Rename to YTEmbed or something on those lines? @sbmadhav : Any suggestion?

Reply:
Hi @vichu1988 , I will do the name changes and update the PR. And for generic video component, the parameters might be different for youtube and  other video hosting websites. Anyways we are creating separate components for each and everything, I would say it would be better to create separate embeds for different hosts to make it simple. Please provide your suggestions / views on this.